theme/index.ftl

```diff
@@ -0,0 +1,23 @@
+[#ftl/]
+[#-- @ftlvariable name="tenant" type="io.fusionauth.domain.Tenant" --]
+[#-- @ftlvariable name="tenantId" type="java.util.UUID" --]
+[#import "_helpers.ftl" as helpers/]
+
+<!doctype html>
+<html>
+  [@helpers.permHeadTag title="Permanent.org - FusionAuth"/]
+  <body class="permanent">
+    [@helpers.permHeader showAdminLock=true/]
+
+    <main style="text-align: center;">
+      <h1>Welcome!</h1>
+      <p>The Permanent web application can be found on <a href="https://permanent.org/app">Permanent.org</a></p>
+      [#-- Maybe put a link to account self service here? --]
+    </main>
+
+    [#-- Using an HTML comment so this is visible in the rendered HTML. We don't want to freak anyone out. --]
+    <!-- Feel free to remove this, it is just a stupid easter egg. Enjoy. -->
+    <!--   "Escape is impossible when you're caught in the net" https://www.imdb.com/title/tt0113957/  -->
+    <div style="position:fixed; left:0; bottom: 0; margin-bottom: 7px; margin-left: 10px;"><a style="color: #fff;" href="https://the-praetorians.net" title="Escape is impossible when you're caught in the net.">π</a></div>
```

Comment:
oh my gosh this is amazing!

Reply:
Ha!!!

So, as a person who very much values fun I will play the potential wet blanket question role: would this element be visible to folks with screen readers?  If so I think we should ditch it.

Reply:
So there is the [ARIA `presentation="none"`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/presentation_role) attribute, but since this is a link that won't work:

> The [`<a>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a) is a special case. Its role would have been exposed even if it had the `presentation` or `none` role directly applied to it. Browsers ignore `role="presentation"` and `role="none"` on focusable elements, including link and inputs, or anything with a [tabindex](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/en-US/docs/Web/HTML/Global_attributes/tabindex) attribute set.Browsers also ignore the inclusion of the role if any of the element contains any global ARIA states and properties, such as [`aria-describedby`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-describedby).

There's the [`aria-hidden` attribute](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-hidden), but that similarly warns:

> Do not use `aria-hidden="true"` on focusable elements.

More generally, they write:

> You should not be hiding visible content unless doing so improves the experience for users of assistive technologies by removing redundant or extraneous content. Only when identical or equivalent meaning and functionality is exposed to assistive technologies can removing visible content from the accessibility API be considered.

I don't know how this applies to easter eggs; the content is rendered but is intentionally zero-contrast to make it hard to discover.

Seems a shame to lose this bit of fun, though. Maybe we only leave it on the dev site, and not on prod? And therefore not check it in here? I don't know.